main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -668,10 +668,10 @@ class Bootstrap(val projectPath: Path, apiKey: Option[String]) {
     // Build artifacts
     println("Building project...")
     val buildResult = buildPkg()
-    buildResult match {
-      case Validation.Success(_) => // Continue
-      case Validation.SoftFailure(_, e) => return Validation.SoftFailure((), e)
-      case Validation.HardFailure(e) => return Validation.HardFailure(e)
+    buildResult.toSoftResult match {
```

Comment:
I think the original code was in error. We want a hard failure here too.

Reply:
Sorry. What I mean is, we can just use toHardFailure and get the same result.

Reply:
Its late. What I mean is, lets use toHardResult and just return HardFailure always. Just like all the other cases.

I will leave it for a subsequent PR.